README.md

```diff
@@ -20,13 +20,43 @@ defp deps do
 end
 ```
 
+For Elixir 1.13 and later, modify `.formatter.exs` by adding
+`Surface.Formatter.Plugin` to `:plugins` and including all Surface files in
+`:inputs`:
+
+```elixir
+# .formatter.exs
+[
+  ...,
+  plugins: [Surface.Formatter.Plugin],
+
+  # add patterns matching all .sface files and all .ex files with ~F sigils
+  inputs: ["lib/**/*.{ex,sface}", ...]
+]
+```
+
+For documentation of other `.formatter.exs` options, see:
+
+- `Surface.Formatter.Plugin` if using Elixir 1.13 or later
+- `mix surface.format` if using Elixir 1.12 or earlier
```

Comment:
should these be links to some other part of the docs?

Reply:
Good question. I assume you probably know these get turned to links in Hex, so I think your point is that these aren't linking if you view the GitHub README on GitHub.

Solid point 👍 I'll explicitly make them links.

Reply:
Actually, after considering the potential wonky edge cases this could result in and taking a look at other Hex packages only to discover this isn't common, I decided let's just leave it alone. If folks want the "full experience" they can go to the docs on Hexdocs. After all, right at the top of the README we say:

> The complete documentation for SurfaceFormatter is located [here](https://hexdocs.pm/surface_formatter/)